addons/medical_treatment/stringtable.xml

```diff
@@ -507,6 +507,14 @@
             <Polish>Pozwala przetoczyć płyny IV samemu sobie</Polish>
             <Italian>Abilita la trasfusione in endovena su se stessi.</Italian>
         </Key>
+        <Key ID="STR_ACE_Medical_Treatment_MedicIV_DisplayName">
+            <English>Allow IV Transfusion</English>
+            <German>Erlaube Bluttransfusionen</German>
+        </Key>
+        <Key ID="STR_ACE_Medical_Treatment_MedicIV_Description">
+            <English>Training level required to transfuse IVs.</English>
+            <German>'Fähigkeiten-Level', das benötigt wird, um Blut zu transfunieren.</German>
```

Comment:
```suggestion
            <German>'Fähigkeiten-Level', das benötigt wird, um Blut zu transfundieren.</German>
```
maybe ust do "transferieren" instead?

Reply:
i'll go with "transfundieren". That seems to be the correct one.